src/hooks/useNfc.ts

```diff
@@ -190,9 +227,14 @@ export const useNfc = () => {
         await NFC.stopRead();
 
         setIsNfcLoading(false);
+        setIsPinConfirmed(false);
+        setIsPinRequired(false);
         if (debitCardData?.status === "OK" && !error) {
           setIsNfcActionSuccess(true);
         } else {
+          if (debitCardData?.status === "ERROR" && !error) {
```

Comment:
If the pin is wrong the response is in the debitCardData object not the error object. Is that a wrong implementation on my side? Should the server send another status then 200? I haven't seen a specification for that in the LUD-3.